src/Gui/ProjectBrowserService.cs

```diff
@@ -83,7 +83,7 @@ public void Load(Project? project)
             {
                 AddComponents(project.Programs);
                 AddComponents(project.MetadataFiles);
-                AddComponents(project.ScriptFiles);
+                AddComponents(project.ScriptFiles.ToList());
```

Comment:
`ConcurrentObserbableCollection.GetEnumerator` calls `ToList` internally, no need to do it here.

Reply:
Done in 140746e